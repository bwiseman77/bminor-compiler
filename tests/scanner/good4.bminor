void function ()
"gimme gimme a \nman \nafter midnight"
5 ? 6 : 7
/* nnooooooo
