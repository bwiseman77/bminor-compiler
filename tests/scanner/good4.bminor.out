-------------------------------------------
           token type  text
-------------------------------------------
                VOID  void
            FUNCTION  function
        LEFT_PARENTH  (
       RIGHT_PARENTH  )
      STRING_LITERAL  gimme gimme a 
man 
after midnight
     INTEGER_LITERAL  5
        TERNARY_PT_1  ?
     INTEGER_LITERAL  6
               COLON  :
     INTEGER_LITERAL  7
             DIVISON  /
      MULTIPLICATION  *
               IDENT  nnooooooo
